crates/factor-variables/build.rs

```diff
@@ -0,0 +1,6 @@
+fn main() {
+    println!("cargo::rerun-if-changed=build.rs");
+    // Enable spin-factors-derive to emit expanded macro output.
+    let out_dir = std::env::var("OUT_DIR").unwrap();
+    println!("cargo:rustc-env=SPIN_FACTORS_DERIVE_EXPAND_DIR={out_dir}");
+}
```

Comment:
Do we have to do this in every factor? Is the only benefit so that you can `CTRL`+`CLICK` rather than run expand macro from the command palette of VS code?

Reply:
The main benefit is that error messages pointing into generated code are actually useful.

Reply:
Now that if you omit the build script you just get normal macro behavior.